docs/creating_component_code_review.md

```diff
@@ -15,11 +15,12 @@ Not all existing code follows the requirements in this checklist. This cannot be
  2. Use existing constants from [`const.py`](https://github.com/home-assistant/core/blob/dev/homeassistant/const.py)
     - Only add new constants to `const.py` if they are widely used. Otherwise keep them on components level
 
-### 1. Requirements
+### 1. External requirements
 
  1. Requirements have been added to [`manifest.json`](creating_integration_manifest.md). The `REQUIREMENTS` constant is deprecated.
  2. Requirement version should be pinned: `"requirements": ['phue==0.8.1']`
  3. We no longer want requirements hosted on GitHub. Please upload to PyPi.
```

Comment:
I think this is not possible anymore

Reply:
Yeah, the 2. should probably also be `Requirement version needs to be pinned`.

Reply:
Oh I was talking about 3

Reply:
Language in 2) updated to "must", and 3) removed.